app/views/pushes/_push.json.jbuilder

```diff
@@ -0,0 +1,33 @@
+json.extract! push, :expire_after_views,
+  :expired,
+  :url_token,
+  :deletable_by_viewer,
```

Comment:
This can be removed for url pushes. But, I haven't change that to make API responses same with API responses of the Premium one.